src/utils/hooks/useDataViews.tsx

```diff
@@ -55,15 +58,21 @@ export const useDataViewsTable = ({ dataViewCode, apiQueries }: DataViewTablePro
   } = useUserState()
 
   useEffect(() => {
-    processRequest(
+    processPostRequest(
       getServerUrl('dataViews', { dataViewCode, query: apiQueries }),
       setError,
       setLoading,
-      setDataViewTable
+      setDataViewTable,
+      filter
     )
   }, [templatePermissions, dataViewCode, apiQueries])
 
-  return { error, loading, dataViewTable }
+  return {
+    error,
+    loading,
+    dataViewTable,
+    filterDefinitions: buildFilterDefinitions(dataViewTable),
+  }
```

Comment:
Had to make the table view a "POST" request, as explained [in back-end](https://github.com/openmsupply/conforma-server/pull/865#discussion_r977182824)

Reply:
Note: I've recombined these back into a single function in the next PR, which is much nicer. This was just done in a hurry.